src/default.js

```diff
@@ -29,6 +29,20 @@ import VueRouter   from 'vue-router';
 Vue.use(VueEvents);
 Vue.use(VueRouter);
 
+// --- Gettext ----
+
+import GetTextPlugin from 'vue-gettext';
+import translations from '../l10n/translations.json'
+
+Vue.use(GetTextPlugin, {
+  availableLanguages: {
+  en_GB: 'British English',
+  de_DE: 'German',
+  },
+  defaultLanguage: 'en_GB',
```

Comment:
Generally speaking we consider en-US the default
the current language is selected from the browser settings. is that already in place?

Reply:
No, currently it's hardcoded english language as default.

I wasn't sure if we should take some language settings from the ownCloud Backend, but for sure, we will respect the browser settings for now.

Reply:
lets start off with the browser setting. THX